modules/virt-cnv-network-alerts.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-events.html/virt-virtualization-alerts.adoc
+
+[id="virt-cnv-network-alerts_{context}"]
+= Network alerts
+
+Network alerts provide information about problems for the CNV Network Operator.
+
+//KubeMacPoolDown Alert
```

Comment:
Redundant leftover?

Reply:
> If you mean the //KubeMacPoolDown thing,
That's the one I was talking about. I just wasn't sure if it's inline documentation, or an outdated part that was commented-out instead of totally removed. Now I understand it's documentation.